service/controller/etcd_backup.go

```diff
@@ -58,7 +59,8 @@ func NewETCDBackup(config ETCDBackupConfig) (*ETCDBackup, error) {
 			NewRuntimeObjectFunc: func() runtime.Object {
 				return new(backupv1alpha1.ETCDBackup)
 			},
-			Name: project.Name() + "-etcd-backup-controller",
+			Name:      project.Name() + "-etcd-backup-controller",
+			SentryDSN: config.SentryDSN,
```

Comment:
Does OperatorKit already have Sentry support or is there something missing from `go.mod`?

Reply:
It had support for sentry since a long time ago.

https://github.com/giantswarm/operatorkit/blob/master/CHANGELOG.md#added-3